example/app/config.yml

```diff
@@ -98,7 +98,7 @@ default:
       username: cumulususer
     services:
       EcsTaskHelloWorld:
-        image: cumuluss/cumulus-ecs-task:1.2.5
+        image: nsidc/cumulus-ecs-task:latest
```

Comment:
Temporary home until we get the PR approved for cumulus-ecs-task and a new version released *and* pushed to devseed's bucket.

Reply:
Why would we host it on devseed's bucket and not update the image for `cumuluss` on Docker hub?

Reply:
@markdboyd  sorry, that comment was based upon a mistaken impression that cumuluss was devseed owned.        I, in fact, mean cumuluss.  I just don't have write access to it, and don't believe most of the team does.

Reply:
I have access to it and I can see if I can grant admin access to other team members if you want

Reply:
💯  thanks